docker-compose.halfstack-ha.yml

```diff
@@ -196,4 +217,16 @@ services:
       --initial-cluster-state $${STATE}"
 
 networks:
-  half:
+  net_redis_node01:
+    driver: bridge
+  net_redis_node02:
+    driver: bridge
+  net_redis_node03:
+    driver: bridge
+  net_sentinel01:
+    driver: bridge
+  net_sentinel02:
+    driver: bridge
+  net_sentinel03:
+    driver: bridge
```

Comment:
Why is this required?

Reply:
I left a comment on the below link about this.
https://github.com/lablup/backend.ai/pull/1567/commits/b626e892a39c5cb4744653804c550800a202abce